schoolopy/main.py

```diff
@@ -129,22 +143,90 @@ def create_building(self, school_id, building):
         """
         return Building(self._post('schools/%s/buildings' % school_id, building.json()))
 
-    def get_users(self):
+    def get_me(self):
+        """
+        Get data of the client using this method (Yourself)
+
+        :return: User object obtained from API. (Of yourself)
+        """
+        return User(self._get('users/me'))
+
+    def get_users(self, inactive=False):
         """
         Get data on all users.
 
+        :param inactive: Gets inactive users instead of normal ones.
         :return: List of User objects.
         """
-        return [User(raw) for raw in self._get('users')['user']]
+        path = 'users' + ('/inactive' if inactive else '')
+        if 'total' not in self._get(path):
```

Comment:
I think the current checks in `_get` handle this, no? If there's an error in the JSON, then `{}` will be returned.